test/wdio/shadow-dom-array/cmp.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('shadow-dom-array', () => {
+  beforeEach(() => {
+    render({
+      template: () => <shadow-dom-array-root></shadow-dom-array-root>,
+    });
+  });
+
+  it('renders children', async () => {
+    await $('shadow-dom-array-root').$('shadow-dom-array').waitForExist();
```

Comment:
No action required - I think this is equivalent to 
```suggestion
    await $('shadow-dom-array-root shadow-dom-array').waitForExist();
```

But might be a bit more performant - https://webdriver.io/docs/api/browser/$/

> Note: chaining $ and $$ commands only make sense when you use multiple selector strategies. You will otherwise make unnecessary requests that slow down the test (e.g. $('body').$('div') will trigger two request whereas $('body div') does literally the same with just one request)